README.md

```diff
@@ -170,29 +170,42 @@ You can create your own card types and add or remove them to or from card number
 ```swift
 struct MyCardType: CardType {
     
-	// MARK: - Required
-	
-	// The name of your specified card type:
+    // MARK: - Required
+
+    // The name of your specified card type:
     public let name = "My Card Type"
-    
+
     // Note: The image that will be displayed in the card number text field's image view when this card type has been detected will load from an asset with the name `cardType.name`.
-	
-	// If the Issuer Identification Number (the first six digits of the entered card number) of a card number 
-	// starts with anything from 1000 to 1111, the card is identified as being of type "MyCardType":
+
+    // If the Issuer Identification Number (the first six digits of the entered card number) of a card number 
+    // starts with anything from 1000 to 1111, the card is identified as being of type "MyCardType":
     public let identifyingDigits = Set(1000...1111)
-	
-	// MARK: - Optional
```

Comment:
When I created a custom card, the CVCLength is required. I think it is an inconsistency in the document so I changed that.